Documentation/metrics.md

```diff
@@ -70,6 +70,8 @@ All these metrics are prefixed with `etcd_network_`
 |---------------------------|--------------------------------------------------------------------|---------------|
 | peer_sent_bytes_total           | The total number of bytes sent to the peer with ID `To`.         | Counter(To)   |
 | peer_received_bytes_total       | The total number of bytes received from the peer with ID `From`. | Counter(From) |
+| peer_sent_faulures_total        | The total number of send failures to the peer with ID `To`.         | Counter(To)   |
```

Comment:
@heyitsanthony should this be `The total number of sending failures`? or sent failures?